api/inbound-stream.js

```diff
@@ -220,40 +220,34 @@ class RedisHandler {
         ]);
 
         // append block hexHeight:dt:id to timeline
+        // append block hexHeight:dt:id to timeline
+        let txInst = _.map(tx.instructions, i => {
+          return [i.program_id, i.keys.join(','), i.data].join('@');
+        }).join('|');
+
         let txnMsg = [
           message.h,
           message.l,
           message.s,
           message.dt,
           message.hash,
-        ];
-        if (tx.instructions.length > 0) {
```

Comment:
I think we need to keep some version of this logic, so that blockexplorer won't crash on solana <=0.12 , or if we make changes to the transaction format in the future